lab-trent/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+
+const errorMiddleware = require('./lib/errormiddleware');
+
+const MONGODB_URI = process.env.MONGODB_URI || 'mongodb://localhost:27017/dev';
+const PORT = process.env.PORT || 3000;
+
+mongoose.Promise = Promise;
```

Comment:
I'd suggest using the bluebird promise implementation, as it's a little more robust than the native node implementation.